back-end/server.js

```diff
@@ -2,6 +2,12 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const http = require('http');
 
+try {
+    require('dotenv').config();
+} catch (e) {
```

Comment:
Lets just use it as an direct dependency instead, this is hideous

Reply:
Yep, fixed. Much better now.